core/container/src/main/java/org/wildfly/swarm/container/runtime/RuntimeDeployer.java

```diff
@@ -234,6 +234,7 @@ public void deploy(Archive<?> deployment, String asName) throws DeploymentExcept
 
             if (BootstrapProperties.flagIsSet(SwarmProperties.EXPORT_DEPLOYMENT)) {
                 final File out = new File(deployment.getName());
+                out.deleteOnExit();
```

Comment:
This looks wrong. It's the very purpose of `swarm.export.deployment` to dump the deployment so that it can be inspected manually. Deleting it on JVM exit defeats the purpose.